pages/team.js

```diff
@@ -267,7 +306,39 @@ and participating in academic competitions with his friends.`}
               />
             </Grid>
             <br />
-            <Text variant="title" color="orange" sx={{ lineHeight: '1.75em' }}>
+            <Text
+              variant="title"
+              color="orange"
+              sx={{ lineHeight: '1.75em', fontSize: ['42px', 6] }}
+            >
+              Bots
+            </Text>
+            <Grid columns={[1, null, 2]} gap={4}>
+              <Bio
+                name="Scrappy"
+                teamRole="Book Keeper"
+                text="The bot that eats trash (and manages #scrapbook)."
+                img="/team/scrappy.png"
+              />
+              <Bio
+                name="Orpheus"
+                teamRole="Assistant"
+                text="The bot that helps the team manage everything."
+                img="/team/orpheus.png"
+              />
+              <Bio
+                name="Toriel"
+                teamRole="Bouncer"
+                text="Just a lil' greeter bot to direct new members in the Hack Club slack channel and collect some analytics data for that process (and Clippy's killer)."
```

Comment:
```suggestion
                text="What a nice lady. She seems kinda lonely. New Hack Club Slack members are onboarded and welcomed through a series of conversations with Toriel."
```
Suggestion: Make Toriel's bio more like her personality.

<img width="377" alt="Screen Shot 2022-05-29 at 00 17 55" src="https://user-images.githubusercontent.com/72365100/170856873-139c690e-38c1-4f1e-a072-de450339886f.png">

Reply:
I love this suggestion, thanks!